2차/류동준_가운데-글자-가져오기.py

```diff
@@ -0,0 +1,7 @@
+def solution(s):
+    if len(s) % 2 == 0:
+        n = int(len(s) / 2)
+        return s[n-1:n+1]
+    else:
+        n = int(len(s) / 2)
+        return s[n]
```

Comment:
```suggestion
    n = int(len(s) / 2)
    return s[n-1:n+1] if len(s) % 2 == 0 else s[n]
```

- n = int(len(s) / 2) 이 코드가 중복으로 들어가기 때문에 위로 분리하는게 좋을 것 같습니다.
- 반환되는 건 len(s) % == 0 값에 따라 달라지기 때문에 하나로 표현하는 것이 좋을 것 같습니다.